appveyor.yml

```diff
@@ -67,32 +61,3 @@ test_script:
   - SET SPEC_OPTS=--format progress
   - SET SLOW=1
   - bundle exec rake
-
-for:
-  -
-    matrix:
-      only:
-        - name: functional-tests-1
-    test_script:
-      - SET SPEC_OPTS=--format progress
-      - SET WORKER_NUMBER=1
-      - SET SLOW=1
-      - bundle exec rake test:functional:windows
-  -
-    matrix:
-      only:
-        - name: functional-tests-2
-    test_script:
-      - SET SPEC_OPTS=--format progress
-      - SET WORKER_NUMBER=2
-      - SET SLOW=1
-      - bundle exec rake test:functional:windows
-  -
-    matrix:
-      only:
-        - name: functional-tests-3
-    test_script:
-      - SET SPEC_OPTS=--format progress
-      - SET WORKER_NUMBER=3
-      - SET SLOW=1
-      - bundle exec rake test:functional:windows
```

Comment:
This is also gonna be a slog... but also worth it in the long run. Thank you.

Reply:
Ya, the times aren't great now but will improve over time.